woocommerce/woocommerce
# Make Multichannel Marketing the default new UI for Marketing page

### All Submissions:


### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/37367.

In this PR, we remove the classic Marketing page and replace it with the new Multichannel Marketing page.

Classic Marketing page that will be removed:

![image](https://user-images.githubusercontent.com/417342/227588170-ad96141b-e3ce-4c0e-bd8f-e8f30d443d88.png)

Multichannel Marketing page:

![image](https://user-images.githubusercontent.com/417342/227588238-8926ada1-a0b9-4e72-8fc7-ea894a07ec7d.png)

The Marketing feature toggle in WooCommerce Settings > Advanced > Features page (`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`) is no longer needed is removed in this PR.

Since the classic Marketing page will not be used anymore, I have also clean up the code base and remove related code that are not in use anymore, i.e. a `Button` component in the `marketing/components` directory.

Some of the code changes here is a reversal of the changes made in PR https://github.com/woocommerce/woocommerce/pull/33964 when we first implemented the feature toggle.

### How to test the changes in this Pull Request:


Test in browser:

1. Go to WooCommerce Settings > Advanced > Features page: `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. You should not see the Marketing feature toggle settings.
3. Go to the marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`
4. Make sure you are seeing the Multichannel Marketing page, not the classic Marketing page.
5. Open the browser console and type `wcAdminFeatures` and press Enter. In the logged object, make sure there is no `marketing-multichannel` property.

![image](https://user-images.githubusercontent.com/417342/227590464-e6dbae2b-f174-4099-a214-6d3ff7745e06.png)



### Other information:




### FOR PR REVIEWER ONLY:
